index.html

```diff
@@ -0,0 +1,225 @@
+            <li><a href="#">Program</a></li>
+            <li><a href="#">Join</a></li>
+            <li><a href="#">Sponsor</a></li>
+            <li><a href="#">News</a></li>
+            <li>
+              <a href="#" class="border border-4 border-danger p-2 text-danger"
+                >AI Campaign</a
+              >
+            </li>
+          </ul>
+        </div>
+      </div>
+    </header>
+    <!-- Main content -->
+    <main class="container-fluid p-0">
+      <!-- Intro section -->
+      <!-- Menu mobile -->
+      <div id="menu" class="hidden-mobile flex-column">
+        <div class="menu-mobile flex-column">
+          <button class="close-btn"><i class="bi bi-x-lg"></i></button>
+          <ul class="menu-options">
+            <li><a href="index.html">Home</a></li>
+            <li><a href="about.html">About</a></li>
+            <li><a href="#">Program</a></li>
+            <li><a href="#">Join</a></li>
+            <li><a href="#">Sponsor</a></li>
+            <li><a href="#">News</a></li>
+            <li>
+              <a href="#" class="border border-4 border-danger p-2 text-danger"
+                >AI Campaign</a
+              >
+            </li>
+          </ul>
+        </div>
+      </div>
+      <img
+        src="img/bg-intro-min.png"
+        alt="AI background image"
+        class="bg-image"
+      />
+      <section class="intro-container d-flex flex-column gap-3 p-3 p-lg-5">
+        <button class="hidden-mobile menu-btn btn align-self-start p-2">
+          <img src="img/btn-menu.png" alt="menu mobile" width="24px" />
+        </button>
+        <span class="fs-3 mt-5 title">"Hello! Human world"</span>
+        <h1 class="mb-3 title fw-bold">AI brains Global Summit 2023</h1>
+        <blockquote
+          class="col-sm-4 col-md-6 col-lg-6 border border-3 border-white fs-6 p-3"
+        >
+          "The best way to predict the future is to invent it." - Alan Kay
+        </blockquote>
+        <span class="fs-3 fw-bold subtitle">2023.04.22(SAT) ~ 23(SUN)</span>
+        <small class="fs-6">@The Artificial Minds Museum</small>
+      </section>
+      <!-- Main Program section -->
+      <section
+        class="main-program p-3 p-lg-5 d-flex flex-column align-items-center gap-3 gap-md-5"
+      >
+        <h2 class="text-white text-center">Main Program</h2>
+        <div class="guide-bar"></div>
+        <div class="cards-container container">
+          <div class="cards row align-items-center gap-1 fs-6 p-1 mt-3">
+            <div class="col-2 p-0 d-flex justify-content-center">
+              <img src="img/program-icon-1.png" alt="Lecture icon" />
+            </div>
+            <h3
+              class="col-4 fw-bold p-0 text-center fs-5 p-md-0 d-flex justify-content-center"
+            >
+              Lecture
+            </h3>
+            <div class="col p-0 text-white">
+              Listen to the speakers from various fields about his toughts.
+            </div>
```

Comment:
- [ ] I think it would be great here if you add the hover animation as shown in the design when a user press on a card it shows a white border so kindly add it to match the design 🔥🔥🔥

I refered to it ⬇⬇
![image](https://user-images.githubusercontent.com/58553711/235039525-5f0b1de5-9807-4b15-8e74-b6bef2844d96.png)